src/components/ListItem.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col'
+import Badge from 'react-bootstrap/Badge';
+
+const ListItem = () => (
+	<Card className="mb-3">
+		<Card.Header as="h5">
+			Sample Component
+		<div className="float-right">
```

Comment:
Is there any other selector we can use other than float?

Reply:
Should be fine in this case. We can use flex alignment later